datasets/sentinel-2/dataset.yaml

```diff
@@ -36,6 +36,6 @@ collections:
             # But we start at depth=2 thanks to the split, so we use a depth of 5 here.
             min_depth: 5
             max_depth: 5
-            chunk_length: 5000
+            chunk_length: 20000
```

Comment:
Increasing the chunk size to decrease the Batch Task parallelism here: we managed to saturate the storage account, I think because this is using a thread pool to walk in parallel.

![image](https://github.com/microsoft/planetary-computer-tasks/assets/1312546/ff49d44d-27b0-4532-a8f4-e9266a5896d9)